pkg/pss/evaluate.go

```diff
@@ -71,7 +79,51 @@ func exemptKyvernoExclusion(defaultCheckResults, excludeCheckResults []pssutils.
 	for _, excludeResult := range excludeCheckResults {
 		for _, checkID := range pssutils.PSS_controls_to_check_id[exclude.ControlName] {
 			if excludeResult.ID == checkID {
-				delete(defaultCheckResultsMap, checkID)
+				for _, excludeFieldErr := range *excludeResult.CheckResult.ErrList {
+					excludeField, _, _, _ := parseField(excludeFieldErr.Field)
+					var excludeBadValues []string
+					switch excludeFieldErr.BadValue.(type) {
+					case string:
+						badValue := excludeFieldErr.BadValue.(string)
+						if badValue == "" {
+							break
+						}
+						excludeBadValues = append(excludeBadValues, badValue)
+					case bool:
+						excludeBadValues = append(excludeBadValues, strconv.FormatBool(excludeFieldErr.BadValue.(bool)))
+					case int:
+						excludeBadValues = append(excludeBadValues, strconv.Itoa(excludeFieldErr.BadValue.(int)))
+					case []string:
+						excludeBadValues = append(excludeBadValues, excludeFieldErr.BadValue.([]string)...)
+					default:
```

Comment:
I don't recommend having an empty default clause. We either remove it or return an error.